src/helpers/augmentedReality/multipleSceneIndexGenerator.ts

```diff
@@ -80,6 +80,14 @@ export const multipleSceneIndexGenerator = ({
     }
 
     localUris = scenes?.[modelIndex]?.localUris;
+
+    if (modelIndex !== 0) {
+      const firstIndex = scenes?.[0]?.localUris;
```

Comment:
the const name is not correct. getting `localUris` from the first index does not result in `firstIndex`. first index makes me think of a `0`. can you find a more precise name here?

Reply:
How about `firstLocalUris`?

Reply:
first of what? maybe more precisely `firstSceneLocalUris`?

Reply:
Yes, it's more understandable.

https://github.com/smart-village-solutions/smart-village-app-app/commit/48355bfc73c3eb1184fd467ccd846dc334c027d7